spec/mock/random-mock.js

```diff
@@ -19,6 +19,12 @@ random.bufferAsync.andCallFake((size) => {
 });
 
 random.randomIdAsync.andCallFake((size) => {
+    if (! size) {
```

Comment:
Add a comment that the real code doesn't do this.  We're adding this check so our tests can help catch scenarios where size can be zero.

... or make it act like the real class.

Throwing is just fine here too.